packages/core/src/transforms/__tests__/resize.spec.ts

```diff
@@ -9,12 +9,15 @@ import { resize } from '../resize'
 
 expect.extend({ toMatchImageSnapshot })
 
-describe('width', () => {
-  let dirCtx: TransformFactoryContext
-  beforeAll(() => {
-    dirCtx = { useParam: vi.fn, warn: vi.fn }
-  })
+let dirCtx: TransformFactoryContext
+beforeAll(() => {
```

Comment:
I had to extract this to the top level, because I got interference between the different `describe` blocks. Not sure what was causing this, but perhaps parallel test running or something similar?

This was the simplest way I found of adressing the issue.